doc/theory/glossary.tex

```diff
@@ -16,56 +79,266 @@
+	
+	\section{Time series}
+	
+	\subsection*{Time series:}
+	A \textit{time series} is a sequence $\{y_i\}_{i = 0}^n$ of real numbers. 
+	
+	A common construction of a times series $\{x_i\}_{i = 0}^n$ is given by choosing $x_0$ arbitrarily as well as a step parameter $h$ and setting
+	\begin{equation*}
+	x_i = x_0 + h\cdot i.
+	\end{equation*}
+	Another usual construction is as follows: given a time series $\{x_i\}_{i = 0}^n \subseteq U$ and a  function
+	\begin{equation*}
+	f :  U \subseteq \mathbb R \to \mathbb R
+	\end{equation*}
+	we obtain a new time series $\{f(x_i)\}_{i = 0.}^n$
+	
+	Generalizing the previous construction we can define a time series from a function 
+	\begin{equation*}
+	\varphi : U  \times M \to M, \qquad U \subseteq \mathbb R, \qquad M \subseteq \mathbb R^d
+	\end{equation*} 
+	using a function $f : M \to \mathbb R$ as follows: let $\{t_i\}_{i=0}^n$ be a time series taking values in $U$, then
+	\begin{equation*}
+	\{f(\varphi(t_i, m))\}_{i=0}^n.
+	\end{equation*}
+	for an arbitrarily chosen $m \in M$.
+	
+	\subsection*{Takens embedding:}	
+	Let $M \subset \mathbb R^d$ be a \underline{compact manifold} of dimension $n$. Let
+	\begin{equation*}
+	\varphi : \mathbb R  \times M \to M
+	\end{equation*} 
+	and
+	\begin{equation*}
+	f : M \to \mathbb R
+	\end{equation*}
+	be generic smooth functions. Then, for any $\tau > 0$ the map
+	\begin{equation*}
+	M \to \mathbb R^{2n+1}
+	\end{equation*}
+	defined by
+	\begin{equation*}
+	x \mapsto\big( f(x), f(x_1), f(x_2), \dots, f(x_{2n}) \big)
+	\end{equation*}
+	where 
+	\begin{equation*}
+	x_i = \varphi(i \cdot \tau, x)
+	\end{equation*}
+	is an injective map with full rank. 
+	\paragraph{\textit{Reference}:} \cite{takens1981detecting}
+	
+	\subsection*{Manifold:} Intuitively, a manifold of dimension $n$ is a space locally equivalent to $\mathbb R^n$. Formally, a subset $M$ of $\mathbb R^d$ is an $n$-dimensional manifold if for each $x \in M$ there exists an 
+	open ball $B(x) = \{ y \in M\,;\ d(x,y) < \epsilon\}$ and a smooth function with smooth inverse 
+	\begin{equation*}
+	\phi_x : B(x) \to \{v \in \mathbb R^n\,;\ ||v||<1\}.
+	\end{equation*}
+	\paragraph{\textit{References}:}  \cite{milnor1997topology,guillemin2010differential}
+	
+	\subsection*{Compact subset:}	A subset $K$ of a metric space $(X,d)$ is said to be \textit{bounded} if there exist a real number $D$ such that for each pair of elements in $K$ the distance between them is less than $D$. It is said to be \textit{complete} if for any $x \in X$ it is the case that $x \in K$ if for any $\epsilon > 0$ the intersection between $K$ and $\{y \,;\ d(x,y) < \epsilon \}$ is not empty. It is said to be \textit{compact} if it is both bounded and complete.
+	
+	
+	\bibliography{bibliography}{}
+	\bibliographystyle{alpha}
+	
+\end{document}
+
+% \subsection*{$L^p$-norm:}The $L^p$ norm of $\mathbb R^n$ is defined for $x = (x_1, \dots, x_n)$ by
```

Comment:
Would it make sense to remove these commented-out lines?

Reply:
All commented lines removed.